components/ui/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,81 @@
+import { cva, VariantProps } from "class-variance-authority";
+import { Menu, Transition } from "@headlessui/react";
+import { Fragment } from "react";
+import { ChevronDownIcon } from "@heroicons/react/20/solid";
+
+const dropDownStyles = cva("", {
+  variants: {
+    items: {
+      primary: "bg-blue-200",
```

Comment:
Minor nit, let's change this to use the `brand` color and `zinc` for the secondary

Reply:
Also, I think we need dark mode styles

Reply:
I see that this is what the previous `resolved` conversation was about :P